x/profiles/types/error.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+// DONTCOVER
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// x/ibc/profiles module sentinel errors
+var (
+	ErrInvalidVersion      = sdkerrors.Register(ModuleName, 1501, "invalid version")
+	ErrMaxProfilesChannels = sdkerrors.Register(ModuleName, 1, "max profiles channels")
```

Comment:
I think we can change the error codes without any problem here: 
```suggestion
	ErrInvalidVersion      = sdkerrors.Register(ModuleName, 1, "invalid version")
	ErrMaxProfilesChannels = sdkerrors.Register(ModuleName, 2, "max profiles channels")
```